packages/relay/src/__tests__/__snapshots__/QueryRenderer.test.js.snap

```diff
@@ -2,8 +2,13 @@
 
 exports[`QueryRenderer renders footerComponent if passed 1`] = `
```

Comment:
Message does not seem to agree with the output?

Reply:
Thanks, I will update the test 😄

Reply:
🚀